src/renderer/components/AircraftSection/VersionHistory.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styled from "styled-components";
+import { shell } from "electron";
+import { ModVersion } from "renderer/components/App";
+
+/**
+ * Container for versions
+ */
+export const Versions = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const GITHUB_RELEASE_BASE_URL = 'https://github.com/flybywiresim/a32nx/releases/tag/';
```

Comment:
**suggestion:** It's better to keep constants in a separate file at the root level.